ui/app/components/ui/check-box/check-box.component.js

```diff
@@ -0,0 +1,34 @@
+import React, { PureComponent } from 'react'
+import PropTypes from 'prop-types'
+import classnames from 'classnames'
+
+export default class CheckBox extends PureComponent {
+  static propTypes = {
+    className: PropTypes.string,
+    checked: PropTypes.bool,
```

Comment:
should there be an indeterminate state?

Reply:
Well, there _is_ a state which is neither "all" or "none", so it would be nice for the checkbox to reflect this. But it's not a priority.

I have an old half-finished checkbox component that I might resurrect that handles indeterminate state properly, and it should be more accessible because it uses a real checkbox instead of a styled div. Maybe we can take this for now, but sub it out for my checkbox component later on when we get the time.